core/test/regression/importer/importer_spec.js

```diff
@@ -809,6 +809,52 @@ describe('Integration: Importer', function () {
                 });
         });
 
+        it('can handle related tags with missing optional fields', function () {
```

Comment:
IMO this should at least mention that the slug field is missing _or_ maybe be explicit about what the missing optional fields are.

Even if you did like

```js
exportData.data.tags[0] = testUtils.DataGenerator.forKnex.createTag();
delete exportData.data.tags[0].slug;
```

Reply:
I like this idea of how explicit deleting a field is! 

Tried out this approach and concluded it generates too much noise because you'd need to:
```
delete exportData.data.tags[0].slug;
delete exportData.data.tags[0].description;
// and so on for each optional field
```

Opted into leaving a comment about 'name' field being the only required field to handle the import correctly.